src/utils/index.ts

```diff
@@ -0,0 +1,5 @@
+// Export all utility functions
+export * from "./common";
+export * from "./file-keys";
+export * from "./toolConfirmation";
```

Comment:
we're not consistent with file naming. i'd prefer this to be caed `tool-confirmation`, but if we want to stick with react conventions, technically `file-keys` should be `FileKeys`, `event-bus` should be `EventBus`, etc